### DISCUSSION TITLE
aggressive blur enabler bootloop fix

### DISCUSSION THREAD

**Discussion by elisenlebkuch** [Mahmud0808/Iconify#343] [Q&A] [answered]
Just would like to know how/if bootlooping would be possible to fix before I enable the "aggressive blur" feature.

**Comment by Mahmud0808** [comment, marked as answer]
Multiple solutions available:
• Boot into [safe mode](https://www.androidauthority.com/how-to-enter-safe-mode-android-801476/)
• Delete Iconify folder from data/adb/modules directory (Magisk)
• Run this command in adb `adb wait-for-device shell magisk --remove-modules` (Magisk)
• Delete modules.img from data/adb/ksu directory (KernelSU)
